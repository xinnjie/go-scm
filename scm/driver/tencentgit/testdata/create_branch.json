{
  "protected": false,
  "developers_can_push": false,
  "developers_can_merge": false,
  "name": "testing",
  "commit": {
    "id": "dc238bca5c226bb7a2c7cc5bdbfc9930d6abd8b6",
    "message": "add README",
    "parent_ids": [],
    "authored_date": "2021-08-20T10:45:48+0000",
    "author_name": "xinnjie",
    "author_email": "wx_1600e990b7cb469a97087b334c94acee@git.code.tencent.com",
    "committed_date": "2021-08-20T10:45:48+0000",
    "committer_name": "xinnjie",
    "committer_email": "wx_1600e990b7cb469a97087b334c94acee@git.code.tencent.com",
    "title": "add README",
    "scroll_object_id": null,
    "created_at": "2021-08-20T10:45:48+0000",
    "short_id": "dc238bca"
  },
  "description": "",
  "branch_type": {
    "name": "Others",
    "color": "#7F8C8D"
  }
}