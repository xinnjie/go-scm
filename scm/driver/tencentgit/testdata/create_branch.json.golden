{
  "Name": "testing",
  "Sha": "dc238bca5c226bb7a2c7cc5bdbfc9930d6abd8b6"
}
